docs/circuit_cutting/explanation/index.rst

```diff
@@ -10,39 +10,185 @@ Circuit cutting is a technique to increase the size of circuits we can run on qu
++------------------------------------------------+-----------------------------------+-------------------------------------------------------------------------+
+| :class:`~qiskit.circuit.library.RXXGate`,      | :math:`(|\theta/2|, 0, 0)`        | :math:`\left[1 + 2 \left|\sin(\theta)\right| \right]^2`                 |
+| :class:`~qiskit.circuit.library.RYYGate`,      |                                   |                                                                         |
+| :class:`~qiskit.circuit.library.RZZGate`,      |                                   |                                                                         |
+| :class:`~qiskit.circuit.library.RZXGate`       |                                   |                                                                         |
++------------------------------------------------+-----------------------------------+-------------------------------------------------------------------------+
+| :class:`~qiskit.circuit.library.CRXGate`,      | :math:`(|\theta/4|, 0, 0)`        | :math:`\left[1 + 2 \left|\sin(\theta/2)\right| \right]^2`               |
+| :class:`~qiskit.circuit.library.CRYGate`,      |                                   |                                                                         |
+| :class:`~qiskit.circuit.library.CRZGate`,      |                                   |                                                                         |
+| :class:`~qiskit.circuit.library.CPhaseGate`    |                                   |                                                                         |
++------------------------------------------------+-----------------------------------+-------------------------------------------------------------------------+
+| :class:`~qiskit.circuit.library.XXPlusYYGate`, | :math:`(|\theta/4|,|\theta/4|,0)` | :math:`\left[1+4\left|\sin(\theta/2)\right|+2\sin^2(\theta/2)\right]^2` |
+| :class:`~qiskit.circuit.library.XXMinusYYGate` |                                   | (independent of :math:`\beta` parameter)                                |
++------------------------------------------------+-----------------------------------+-------------------------------------------------------------------------+
+| :class:`.Move` (cut wire) without classical    | not applicable                    | :math:`4^2=16`                                                          |
+| communication (i.e., in the LO setting)        |                                   |                                                                         |
++------------------------------------------------+-----------------------------------+-------------------------------------------------------------------------+
 
 Current limitations
 -------------------
 * ``PauliList`` is the only supported observable format until no sooner than CKT v0.7.0.
```

Comment:
I think we can remove this after merging #523

Reply:
Yup, I was planning to change this line once that merged.  Fixed in 3c3d23945732addc8327219b04a255d756a454e8.